<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Vishnus-MacBook-Pro.local" ignored="0" tests="4" failures="1" timestamp="07 Mar 2019 23:55:47 GMT" time="0.042" errors="0">
  <testcase name="ast3" time="0.002" classname="Test2">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]
at Test2.ast3(Test2.java:24)
... Removed 41 stack frames]]>
    </failure>
  </testcase> <!-- ast3 -->
  <testcase name="test" time="0.001" classname="Test2"/>
  <testcase name="test2" time="0.0" classname="Test2"/>
  <testcase name="test" time="0.0" classname="Test3"/>
</testsuite>
